src/codeql.ts

```diff
@@ -283,6 +284,11 @@ export const CODEQL_VERSION_BETTER_RESOLVE_LANGUAGES = "2.10.3";
  */
 export const CODEQL_VERSION_SECURITY_EXPERIMENTAL_SUITE = "2.12.1";
 
+/**
+ * Versions 2.12.3+ of the CodeQL CLI support the `--qlconfig` flag in calls to `database init`.
+ */
+export const CODEQL_VERSION_INIT_WITH_QLCONFIG = "2.12.3";
```

Comment:
```suggestion
export const CODEQL_VERSION_INIT_WITH_QLCONFIG = "2.12.4";
```

We missed the window for 2.12.3. Will be in 2.12.4.